apps/clinic/src/modules/appointments/appointments.service.ts

```diff
@@ -17,14 +22,26 @@ export class AppointmentsService {
     private appointmentsRepository: AppointmentsRepository,
     private doctorsService: DoctorsService,
     private patientsService: PatientsService,
+    @Inject('CLINIC_KAFKA_CLIENT') private readonly client: ClientKafka,
   ) {}
 
   async createAppointment(
     dto: AppointmentDto,
     user: JwtPayload,
   ): Promise<AppointmentEntity> {
     const patient = await this.patientsService.getPatient(user);
-    return this.appointmentsRepository.createAppointment(dto, patient.id);
+    const payload = await this.appointmentsRepository.createAppointment(
+      dto,
+      patient.id,
+    );
+    const doctor = await this.doctorsService.getDoctorById(payload.doctorId);
+
+    this.client.emit(EventsEnum.clinicAppointmentCreated, {
```

Comment:
this.appointmentPublisher.create({ type: NotificationsEnum.appointment,
      userId: doctor.userId,
      payload})